packages/hebao_v1/contracts/iface/SubAccount.sol

```diff
@@ -24,12 +24,10 @@ contract SubAccount
     /// @dev Deposits Ether/token from the wallet to this sub-account.
     ///      The method must throw in case of error, or must emit a SubAccountTransfer event.
     /// @param wallet The wallt from which the Ether/token will be transfered out.
-    /// @param signers The list of meta-transaction signers, must be emptpy for normal transactions.
```

Comment:
I guess by removing signers here it could kind of limit the more riskier apps of making use of `SubAccount`. Though for ERC20 this could be solved quite nicely by not approving anything in the module itself and be counting on `approveToken` in `ApprovedTransfers` (but this doesn't work with ETH). Another possibility is by enforcing the quota. So there are definitely some options even in those (probably pretty rare) cases.

Reply:
The idea I have is that those dApps modules can only be accessed by the owners, not guardians or any other roles that a wallet may have. Therefore all meta-transactions must be signed by and only by owners.

Reply:
The reason I think the signers would be helpful is for more security. There's a risk if the owner keypair gets stolen and the hacker can use it to extract all wallet funds without any limits just with the owner keypair. But probably all dapps we will support in the modules will have limited risks (and otherwise other solution can probably be found).

Reply:
I'll think about it. Requiring other signers means great user experience problems.